src/arrayMethodSort.js

```diff
@@ -4,8 +4,18 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b),
```

Comment:
The default `compareFunction` should return a number, not a boolean. The comparison `(a, b) => String(a) > String(b)` returns a boolean, which is not suitable for sorting. Consider using `(a, b) => String(a).localeCompare(String(b))` instead.